browser/resources/resource_ids

```diff
@@ -1,4 +1,4 @@
-# Resource ids starting at 31000 are reserved for projects built on Chromium.
+# Resource ids starting at 38000 are reserved for projects built on Chromium.
```

Comment:
Is this file auto-generated?

Reply:
No, it's updated manually.

Reply:
oh wow; do you update the number here too?

Reply:
I just updated the comment.
Actual number shifting was done by https://github.com/brave/brave-core/commit/de534390b4b0043c0cdcefd9cd73306e5315a776

Reply:
ahh - thanks! 😄